src/models/db/transfers.js

```diff
@@ -90,6 +90,20 @@ function * getTransfer (uuid, options) {
     })
 }
 
+function * getTransferByExecutionCondition (executionCondition, options) {
```

Comment:
Looks like a good case for extracting a method to reduce duplication between this method and `getTransfer`